source/js/components/paypal.js

```diff
@@ -13,9 +13,12 @@ export default function initPaypal(
   var loadingErrorMsg = window.gettext(
     "An error occurred. Please reload the page or try again later."
   );
-  var generalErrorMsg = window.gettext(
+  var authErrorMsg = window.gettext(
     "There was an error processing your payment. Please try again."
   );
+  var generalErrorMsg = window.gettext(
+    "Unable to connect to Paypal. Please try again in a few minutes."
```

Comment:
Tiny typo

```suggestion
    "Unable to connect to PayPal. Please try again in a few minutes."
```

Reply:
Great catch @TheoChevalier! I have updated it to reflect your feedback